dali/test/python/test_RN50_external_source_parallel_train_ddp.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This file is marked as executable, but doesn't specify the interpreter.

Reply:
Fixed